src/Application/ConsoleApplication.php

```diff
@@ -59,41 +63,33 @@ private function handleCommand(string $commandName): void
 
         $handler = $consoleCommandConfig->handler;
 
-        $params = $this->resolveParameters($handler);
+        try {
```

Comment:
We have two failing tests at the moment - this is because we're missing type-inferred validation (#229)